recipes/liburing/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set version = "2.5" %}
+
+package:
+  name: liburing
+  version: {{ version }}
+
+source:
+  url: https://github.com/axboe/liburing/archive/liburing-{{ version }}.tar.gz
+  sha256: 456f5f882165630f0dc7b75e8fd53bd01a955d5d4720729b4323097e6e9f2a98
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage('liburing', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+
+test:
+  commands:
+    - test -f "$PREFIX/include/liburing.h"
+    - test -f "$PREFIX/lib/liburing.a"
```

Static libs should not be in the main package 
```suggestion
    - test ! -f "$PREFIX/lib/liburing.a"
```

---

Thanks, I took inspiration from [libuv-feedstock](https://github.com/conda-forge/libuv-feedstock/blob/25d64e3244ede518b02a546e4c262dd103df78e8/recipe/meta.yaml#L33), do they do it wrong there too?

---

Yes. That is also wrong.

---

Indeed, they have an issue: https://github.com/conda-forge/libuv-feedstock/issues/63.